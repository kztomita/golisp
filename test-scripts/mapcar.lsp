
(print (mapcar #'car '((1 a) (2 b) (3 c))))
(print (mapcar #'cons '(a b c) '(1 2 3)))
(print (mapcar #'(lambda (x) (* x 2 )) '(1 2 3)))
