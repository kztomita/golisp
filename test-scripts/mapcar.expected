
(1 2 3) 
(1 2 3) 
((a . 1) (b . 2) (c . 3)) 
(2 4 6) 
